service/controller/release/resource/apps/resource_test.go

```diff
@@ -1,132 +0,0 @@
-package apps
-
-import (
-	"sort"
-	"strconv"
-	"testing"
-
-	appv1alpha1 "github.com/giantswarm/apiextensions/v2/pkg/apis/application/v1alpha1"
-	releasev1alpha1 "github.com/giantswarm/apiextensions/v2/pkg/apis/release/v1alpha1"
-	"github.com/google/go-cmp/cmp"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
-	"github.com/giantswarm/release-operator/v2/service/controller/key"
-)
-
```

Comment:
Was it a conscious decision to not port tests to the new implementation?

Reply:
They will be ported. Getting remarks on code was a priority.